Source/WebCore/rendering/svg/RenderSVGTransformableContainer.cpp

```diff
@@ -94,7 +94,7 @@ void RenderSVGTransformableContainer::updateLayerTransform()
 
 void RenderSVGTransformableContainer::applyTransform(TransformationMatrix& transform, const RenderStyle& style, const FloatRect& boundingBox, OptionSet<RenderStyle::TransformOperationOption> options) const
 {
-    applySVGTransform(transform, graphicsElement(), style, boundingBox, m_supplementalLayerTransform.isIdentity() ? std::nullopt : std::make_optional(m_supplementalLayerTransform), options);
+    applySVGTransform(transform, graphicsElement(), style, boundingBox, std::nullopt, m_supplementalLayerTransform.isIdentity() ? std::nullopt : std::make_optional(m_supplementalLayerTransform), options);
```

Comment:
A helper lambda here could make this line more readbale.

Reply:
Done.